src/Chapter2.hs

```diff
@@ -500,7 +502,8 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
+isThird :: [Int] -> Bool
+isThird list = if (take 1 (reverse (take 3 list))) == [42] then True else False
```

Comment:
Wow that is a creative implementation 🎨 

Nothe that you don;t need to put in into `if-then-else` as it already returns `Bool`.
```suggestion
isThird42 list = take 1 (reverse (take 3 list)) == [42] 
```

And, btw, tests are failing because they expect this function to be named `isThird42` 🙂

Reply:
On the other hand, here you could just use the pattern matching. It would be less expensive 🙂 

```haskell
isThird42 (_ : _ : 42 : _ ) = True
isThird42 _ = False
```